src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1473,7 +1473,6 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Predefined type '{0}' is not defined or imported</value>
   </data>
   <data name="ERR_PredefinedValueTupleTypeNotFound" xml:space="preserve">
-    <!-- We need a specific error code for ValueTuple as an IDE codefix depends on it (AddNuget) -->
```

Comment:
Why remove this comment?

Reply:
It was removed by VS. I'll put it on the ErrorCode enum instead, as people almost never edit this file by hand (rewritten by VS when you edit it in the designer).

---
In reply to: [162499187](https://github.com/dotnet/roslyn/pull/24199#discussion_r162499187) [](ancestors = 162499187)

Reply:
Thanks